erpnext/public/js/controllers/buying.js

```diff
@@ -195,6 +195,10 @@ erpnext.buying.BuyingController = erpnext.TransactionController.extend({
 		this._super(doc, cdt, cdn);
 	},
 
+	batch_no: function(doc, cdt, cdn) {
+		this._super(doc, cdt, cdn);
+	},
```

Comment:
As we are using this feature for selling side, no need to call this from the buying.js file